src/core/config/mac.h

```diff
@@ -489,16 +489,25 @@
 #endif
 
 /**
- * @def OPENTHREAD_CONFIG_CSL_MIN_RECEIVE_ON
+ * @def OPENTHREAD_CONFIG_MIN_RECEIVE_ON_AHEAD
```

Comment:
The name should have "CSL" in it, I think - since the file contains any MAC related configurations, not only CSL ones.

Reply:
I removed it on purpose since the definition applies for any reception, not only CSL. Maybe there would be another non-CSL related use case for this value in the future?

Reply:
Right, that can be for non-CSL cases as well. But isn't this then simply a constant defined by 802.15.4 instead of something the platform can configure? Like other 802.15.4 constants we use.
A radio that's ready to receive at StartMhrTime - 192 us will be able to receive the full frame always (given that no errors occur after this time).

Reply:
See comment below.